functions/src/services/trigger/triggerService.ts

```diff
@@ -152,6 +152,20 @@ export class TriggerService {
         `TriggerService.userEnrolled(${user.id}): Adding welcome message failed due to ${String(error)}`,
       )
     }
+
+    try {
+      await this.factory.message().addMessage(
+        user.id,
+        UserMessage.createSymptomQuestionnaire({
```

Comment:
I added a check to make sure only to add a questionnaire response, if the enrollment date is within 24 hours of the enrollment date modulo 14 days, and that the duration since enrollment is higher than 24 hours, so there should not be duplicate messages - there is however no test so far